Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1

```diff
@@ -250,50 +250,58 @@ Set-Alias -Scope Global Rebuild-Solution Invoke-RebuildSolution
 Function Invoke-RebuildSolution {
     Param(
         [string] $buildConfiguration = "Debug",
-        [string] $target = "build",
         [string] $verbosity = "minimal",
-        [string] $maxCpuCount = "",
-        [string] $noReuse = "false",
-        [string] $platformEnvironmentVariable = "Any CPU",
-        [string] $solutionPath = (Get-RepositoryResolvedPath "Application/Ed-Fi-Ods.sln"),
-        [string] $msBuildFilePath = $env:msbuild_exe
+        [string] $solutionPath = (Get-RepositoryResolvedPath "Application/Ed-Fi-Ods.sln")
     )
     Invoke-Task -name $MyInvocation.MyCommand.Name -task {
         if ((Get-DeploymentSettings).Engine -eq 'PostgreSQL') { $buildConfiguration = 'Npgsql' }
         if (-not [string]::IsNullOrWhiteSpace($env:msbuild_buildConfiguration)) { $buildConfiguration = $env:msbuild_buildConfiguration }
 
-        $msBuildParameters = @(
-            "`"/property:Configuration=$buildConfiguration`"",
-            "`"/target:$target`"",
-            "`"/verbosity:$verbosity`"",
-            "`"/nr:$noReuse`"",
-            "`"/p:Platform=$platformEnvironmentVariable`""
-        )
-
-        if (-not [string]::IsNullOrWhiteSpace($maxCpuCount)) {
-            $msBuildParameters += "`"/m:$maxCpuCount`""
-        }
-        else {
-            $msBuildParameters += "`"/m`""
-        }
-
         $params = @{
             Path                           = $solutionPath
-            MsBuildParameters              = ($msBuildParameters -join ' ')
-            ShowBuildOutputInCurrentWindow = $true
+            BuildConfiguration             = $buildConfiguration
             LogVerbosityLevel              = $verbosity
         }
 
-        if (-not [string]::IsNullOrWhiteSpace($msBuildFilePath)) { $params.MsBuildFilePath = $msBuildFilePath }
-
         ($params).GetEnumerator() | Sort-Object -Property Name | Format-Table -HideTableHeaders -AutoSize -Wrap | Out-Host
-        $buildResult = Invoke-MsBuild @params
 
-        if ($null -eq $buildResult.BuildSucceeded) { throw "Unsure if build passed or failed: $($buildResult.Message)" }
-        if ($buildResult.BuildSucceeded -eq $true) { return }
-        if ($buildResult.BuildSucceeded -eq $false) {
+		# Local Variables.
```

Comment:
please de-indent once on lines 268 to 293.

Reply:
Fixed